tests/test_logsub.py

```diff
@@ -32,7 +32,8 @@ def setUp(self):
             json.dump(self.config_data, conf)
 
     def tearDown(self):
-        os.remove('config.json')
+        if os.path.isfile('config.json'):
```

Comment:
I think this fixes the occasional/random test failure that comes from test_logsub.py